LocalSettings.php

```diff
@@ -477,6 +477,9 @@
 		'obeymewiki' => '/srv/mediawiki/w/skins/chameleon/layouts/fixedhead.xml',
```

Comment:
Wiki overrides should be removed here now. Keeping only default.

Reply:
Done. Is there a way to let those wikis know or set the option for them?

Reply:
> Done. Is there a way to let those wikis know or set the option for them?

We can set them once merged.